README.md

```diff
@@ -45,6 +45,15 @@ If you want to use the `library` trait, in order to e.g. make a bliss plugin for
 and build the examples associated with it, you will also need `libsqlite3-dev` on Ubuntu, or `sqlite` on
 Archlinux.
 
+Note: it is also possible to use the [Symphonia](https://github.com/pdeljanov/Symphonia)
+crate instead of ffmpeg to handle the decoding of songs. See the
+[Decoders](#decoders) section below for more information.
+
+However, it makes bliss-rs easier to package, since it removes a big dependency to ffmpeg.
```

Comment:
that last line seems a little out of place, maybe:

```suggestion
Note: it is also possible to use the [Symphonia](https://github.com/pdeljanov/Symphonia)
crate instead of ffmpeg to handle the decoding of songs. See the
[Decoders](#decoders) section below for more information.

The Symphonia decoder is slightly slower, but using it make bliss-rs easier to package, since it removes a big dependency to ffmpeg.
```

would flow better?